### DISCUSSION TITLE
I want to see all detected programs although they are updated or not.

### DISCUSSION THREAD

**Discussion by gurayAras** [NexovaDev/UpdateHub#430] [Ideas]
Hi,

I want to know that all detected programs so I can understand which programs are not detected by UpdateHub.

If this feature added to app, I would really appreciated.

**Comment by onyx-nxt** [comment]
Seen this feature requested a lot, where would we show this list? In the search page?

**Comment by gurayAras** [comment]
Hi,

Thank you for your response. 

I think if it is feasible, it would be nice when we enter the Apps section we can see all the installed programs on our computer.

Thanks.

**Comment by Bartolomeus383** [comment]
I have only recently installed Updatehub and had not yet been able to deploy it effectively because of the 'timeout reached' error. Now that this error seems to have been resolved, I am only just finding out that there is simply no list of 'monitored software'. 
It seems obvious to me that you need to know which packages are, and which are not, monitored for updates. So I agree with the request to implement this option, in fact it also seems obvious to me that this is a prerequisite to using an updater at all, and I am honestly very surprised that it is not there yet. I have yet to come across a software updater that cannot produce a list of monitored programs. 
I am, however, pleased that this discussion is there, that it is being responded to positively, and that I witnessed the "timeout reached" problem being resolved so quickly. 

About the question of where to show the program list: Just make a button with 'installed programs', or better yet 'monitored programs' in the row with the buttons 'dashboard', 'recently updated' etc.
